packages/capabilities/src/usage.js

```diff
@@ -40,3 +40,22 @@ export const report = capability({
     )
   },
 })
+
+/**
+ * Capability can be invoked by an agent to record usage data for a given resource.
+ */
+export const record = capability({
+  can: 'usage/record',
+  with: SpaceDID,
+  nb: Schema.struct({
+    /** MailTo DID of the customer that is being billed. */
+    customer: AccountDID,
```

Comment:
question: will this be available in Freeway where this invocation will be created? If I remember correctly I think we may only have information about the space that was authorized, and the handler for this will need to determine who to bill based on that

Reply:
Yeah, I've discussed this with @Peeja and it will be available in Freeway at the time the invocation is triggered, here is the convo: https://github.com/storacha/RFC/pull/36#discussion_r1790818236

Reply:
I think that conversation is missing the step that I'm talking about - @Peeja is right that we'll get the space DID from the content commitment, but there is an additional database query needed to go from space DID to account DID that cannot happen in Freeway (since it doesn't have access to Dynamo, and would take too long anyway)

Reply:
yep! and the space DID should already be available in the `with` so you don't need to add that. thanks!

Reply:
@travis The Provider DID returned by that function is `did:web:string`, but the customer DID is type `AccountID` = `did:mailto:...`. Any suggestions on how to convert it to a mailto DID?

Reply:
ack, sorry, I think I mislead you - apologies! I think the function I linked must be returning the list of "providers" (ie, services) a space is registered to, not the list of "customers" of those providers, which is what you want - let me know if you're unable to find that function, but from Discord I think you're already on it